salt/cloud/clouds/proxmox.py

```diff
@@ -557,21 +557,15 @@ def _reconfigure_clone(vm_, vmid):
         log.warning("Reconfiguring clones is only available under `qemu`")
         return
 
-    # TODO: Support other settings here too as these are not the only ones that can be modified after a clone operation
     log.info("Configuring cloned VM")
 
+    # Get all properties that can be passed to config
+    props = _get_properties("nodes/{node}/qemu/{vmid}/config", "POST")
+
     # Modify the settings for the VM one at a time so we can see any problems with the values
     # as quickly as possible
     for setting in vm_:
-        if re.match(r"^(ide|sata|scsi)(\d+)$", setting):
-            postParams = {setting: vm_[setting]}
-            query(
-                "post",
-                "nodes/{}/qemu/{}/config".format(vm_["host"], vmid),
-                postParams,
-            )
-
```

Comment:
Hi @scornelissen85!  Are you able to elaborate on why this check is no longer needed?

Reply:
Because these settings are now in `props`. This var now contains the valid possible settings retrieved from de api documentation, like those in the regex. I'm now validating this on line 593, with validating if `setting` is in `props`. 

With this change it's not only possible to set `ide0`, `sata0` or `scsi0`, but also `memory` and `cores` and other options that can be set via the Proxmox API (see https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/qemu/{vmid}/config under `POST` for valid settings). It also contains the `net[0-9]` settings. Because of that I had to move line 568 (`if re.match(r"^net(\d+)$", setting):`) as first if-statement. Otherwise the `net[0-9]` settings was not correctly set anymore.

I also have updated the tests and when the API documentation returned the correct properties it will work still as it was with the regex.